### ISSUE TITLE
Unclear phpArgs options in README

### ISSUE THREAD

**Issue by NoMan2000** [jgable/grunt-phplint#7]
It's not clear what the options being passed to php -l are.  The command that I want to run is this with the path to the error files.

php -n -l -d display_errors -d display_startup_errors

I actually think that should be the default for this plugin since it removes any problems with configuration files and strictly dumps the output.  With the current readme file like this:

      phpArgs: {
                "-l": null
      },

That doesn't explain if the extra arguments are to the right or to the left of the colon or what format they should be in.

**Comment by jgable** [comment]
It's been awhile since I had to run phplint, so I'll take your word for it on what the default arguments should be.  If you want to put together a pull request to make that happen, I'd be happy to review and merge it.

For the phpArgs, you are right about the README not being clear about the structure.  In your case the following structure would probably do the trick.  But, I'm not sure how I handle the `-d`switch with multiple values.

```js
phpArgs: {
  "-n": null
  "-l": null
  "-d": ["display_errors", "display_startup_errors"]
},
```

I'll see if I have some time to update it over the next week or so.  Otherwise, you're free to contribute that clarification.

**Comment by NoMan2000** [comment]
Anytime I have to go back to old code, I think of this picture:

http://www.chriszacny.com/static/img/blog_posts/intro_devops_package/gandalf.jpg

**Comment by NoMan2000** [comment]
I updated the readme file in a pull request, but until you're sure on passing an array to -d, I wouldn't merge it in.

**Comment by jgable** [comment]
Closed via #8.
